Draft blog post. (#51)
hpc-carpentry/hpc-carpentry.github.io: _posts/2021/12/2021-12-09-sc21-summary.md

## The BoF at SC21

The HPC Carpentry team hosted a "Birds of a Feather" 
session at the [SC21][sc21] conference.

The primary purpose of this was to try to reconnect with 
the HPC user community, and gather feedback and guidance
on how to prioritize our efforts moving forward.

We imagine that the SC21 audience is probably somewhat more
HPC-aware than the Carpentries community at large, and 
possibly also biased towards facility operators, rather 
than novice users, so the feedback we gained, while obviously
valuable, nevertheless does have this basic bias baked
into it.

This was a hybrid conference, and the BoF sessions were

> done via the Zoom teleconferencing tool, with Q&A done

Reviewer: ```suggestion
done via Zoom, with Q&A done
```

via the Sli.do tool.  The SC21 organizers provided access to
all of these venues for registered attendees.

Andrew Reid was on-site and hosted the on-site portion from 

> the podium, while Trevor Keller, Anajian Alim Rasel, and 
> Alan O'Cais were on-line, monitoring the Zoom and Sli.do 

Reviewer: ```suggestion
the podium, while Trevor Keller, Annajiat Alim Rasel, 
Alan O'Cais, and Wirawan Purwanto were on-line,
monitoring the Zoom and Sli.do 
```

Author: Left off Wirawan because he was not listed as a co-host, but you're right that he was in there doing stuff, which also counts.

questions and keeping the session flowing for all participants.

There were a number of survey questions planned in advance,
which were presented via the polling functionality in the 
Sli.do.  This information was captured and is available
in the [coordination repo][coordrepo].

The principal results are that attendees would like to see,
in descending order of preference, future lessons on containers, 
Python and MPI, Dask, and Julia, would like to be able to 
mix and match lesson content, to construct half-day lessons 
that use the templating capability to adapting lessons to 
permanent on-premise facilities.

After the planned polling questions were done, a more general
discussion ensued, largely captured in a [CodiMD][bofcodi] 
document.

A number of attendees were interested in how to give feedback
on their experience with the lessons, and seemed uncertain what
feedback we're looking for.  The answer is that we are mainly
looking for actionable feedback that helps improve the lessons
for the community.  The main mechanism by which we imagine this 

> happening is through insructor and learner comments.  At the 

Reviewer: ```suggestion
happening is through instructor and learner comments.  At the 
```

end of the lesson, following usual Carpentries practice, organizers
should ask learners for positive and negative feedback, on green
or red sticky notes, and conduct a post-workshop survey.  This
info, particularly the red stickies, is what we're looking for,
and ideally would be translated into issues on the appropriate

> lesson repostiory.

Reviewer: ```suggestion
lesson repository.
```

We will be digesting this feedback over the next few months,
and look forward to opening (and closing!) several issues to 
ensure the lessons benefit from this effort.

> [sc21](https://sc21.supercomputing.org/)
> [coordrepo](https://github.com/hpc-carpentry/coordination/tree/main/conferences/SC21)
> [bofcodi](https://codimd.carpentries.org/9-Y8OaVIT2qpb_P47TR7Lw)

Reviewer: ```suggestion
[sc21]: https://sc21.supercomputing.org/
[coordrepo]: https://github.com/hpc-carpentry/coordination/tree/main/conferences/SC21)
[bofcodi]: https://codimd.carpentries.org/9-Y8OaVIT2qpb_P47TR7Lw
```
